logic/broadcaster.js

```diff
@@ -96,15 +96,25 @@ Broadcaster.prototype.bind = function (peers, transport, transactions) {
  * Calls peers.list function to get peers.
  * @implements {modules.peers.list}
  * @param {Object} params
+ * @param {number} params.limit[=self.config.peerLimit] - maximum number of peers to get
+ * @param {string} params.broadhash[=null] - broadhash to match peers with
+ * @param {boolean} params.matchBroadhash[=false] - if true: get only peers with broadhash equal to params.broadhash
+ * @param {boolean} params.unmatchBroadhash[=false] - if true: get only peers with broadhash different than params.broadhash
  * @param {function} cb
  * @return {setImmediateCallback} err | peers
  */
 Broadcaster.prototype.getPeers = function (params, cb) {
 	params.limit = params.limit || self.config.peerLimit;
```

Comment:
`self.config.peerLimit` is confusing, it not come from `config.broadcasts`, but comes from `constants.maxPeers` - it should be removed and we should use `constants.maxPeers` explicitly. Also not need to modify `params.limit` here - `modules.peers.list` will set it for you.

Reply:
1. Reassign the limit from `config.broadcasts` - very good idea
2. Not mutating the input parameter - also a good idea but not a part of this PR.